.github/workflows/test.yml

```
@@ -0,0 +1,15 @@
+name: Test
+on: [pull_request]
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Use Node.js
+      uses: actions/setup-node@v1
+      with:
+        node-version: '12.x'
+    - run: npm install
+    - run: npm test
```

If we use `npm ci` instead of `npm install` then we also get npm to check that the `package-lock.json` is correct. Without using `npm ci` we run the risk of the `package-lock.json` falling out of sync with the `package.json` file.

We could either do the install and test in separate commands or do them in a single command which npm has called `npm cit`

```suggestion
    - run: npm cit
```